docker-compose-dev.yml

```diff
@@ -74,7 +75,7 @@ services:
     links:
       - postgresql
       - redis
-      - elasticsearch
```

Comment:
The main issue that I have here is that the production docker compose has not been tested. So far, only the development compose has been tested, and I thought it would be better to make sure that one works before moving to the production.

Another approach would be to merge both into a single docker compose, and decide if it is dev or prod based on environment variables, for instance. That would simplify adding changes that apply to both infrastructures.

Reply:
Ok, done and tested. Let me know what you think